src/content/configuration/dev-server.md

```diff
@@ -53,7 +53,7 @@ Content not from webpack is served from /path/to/dist/
 
 that will give some background on where the server is located and what it's serving.
 
-If you're using dev-server through the Node.js API, the options in `devServer` will be ignored. Pass the options as a second parameter instead: `new WebpackDevServer(compiler, {...})`. [See here](https://github.com/webpack/webpack-dev-server/tree/master/examples/api/simple) for an example of how to use webpack-dev-server through the Node.js API.
+If you're using dev-server through the Node.js API, the options in `devServer` will be ignored. Pass the options as a second parameter instead: `new WebpackDevServer(compiler, {...})`. [See here](https://github.com/webpack/webpack-dev-server/tree/master/examples/api/simple) for an example of how to use webpack-dev-server through the Node.js API. **Note:** You cannot use the second compiler argument (a callback) when using `WebpackDevServer`.
```

Comment:
If i am reading correct, it says: `Pass me two arguments: first one is compiler, second is object with props`. What exactly should this mean in that case? "You cannot use the second compiler argument (a callback) when using `WebpackDevServer`."

Reply:
When using `WebpackDevServer` you pass it the compiler.

But if you pass 2 args into the webpack compiler (as in `new WebpackDevServer(webpack(first, second))`, it gives a cryptic error.

Reply:
Looks like the CI build fails now when running since this PR's been sitting here for a while.

Was my reasoning incorrect? I just want to help other developer who'll understandably end up in the same situation.

Reply:
```
If you don’t pass the webpack runner function a callback, it will return a webpack Compiler instance. This instance can be used to manually trigger the webpack runner or have it build and watch for changes, much like the CLI.
```

Reply:
The linked example also doesnt pass the runner function, so thats ok.
Based on this, i'd suggest to update your changes to:
```
If you're using .......... Node.js API. Remember to [avoid providing a runner function to `webpack()`](/api/node/#compiler-instance) as a second argument in order to get a `compiler` instance.
```